src/media/css/app-list.styl

```diff
@@ -40,10 +40,28 @@ $desktop-tile-padding = 20px 50px;
     list-style: none;
     margin-top: 30px;
 
-    li:not(.loadmore) {
+    > li:not(.loadmore) {
         border-bottom: 1px solid $filters-border;
+        padding-bottom: 15px;
         padding-top: 15px;
     }
+    // Masking element to cover the last row's bottom border.
+    &:after {
```

Comment:
We could save a lot of CSS and maintenance if we remove app list borders from scope. Collection landing doesn't have borders so itd be more consistent.